### PR TITLE
Python 3 support #5

### PR DESCRIPTION
**Pull request by Liwink**
#5

**py.test-3.5 all passed**

```
$ py.test-3.5
============================================================ test session starts ============================================================
platform darwin -- Python 3.5.1, pytest-2.8.7, py-1.4.31, pluggy-0.3.1
rootdir: /Users/Liwink/projects/mongokat, inifile:
plugins: cov-2.2.1, html-1.10.0
collected 58 items

test_api.py ......
test_collection.py .
test_ensure_fields.py .
test_hooks.py ..
test_mongokit_api.py .s.............s..s.ss..s..sss.ssss...sssss..s
test_readme_sample.py .
test_shortnames.py .

=================================================== 39 passed, 19 skipped in 0.77 seconds ===================================================
```

**py.test-2.7 all passed**

```
$ py.test-2.7
============================================================ test session starts ============================================================
platform darwin -- Python 3.5.1, pytest-2.8.7, py-1.4.31, pluggy-0.3.1
rootdir: /Users/Liwink/projects/mongokat, inifile:
plugins: cov-2.2.1, html-1.10.0
collected 58 items

test_api.py ......
test_collection.py .
test_ensure_fields.py .
test_hooks.py ..
test_mongokit_api.py .s.............s..s.ss..s..sss.ssss...sssss..s
test_readme_sample.py .
test_shortnames.py .

=================================================== 39 passed, 19 skipped in 0.51 seconds ===================================================
```

### CODE REVIEW

**Comment by sylvinus** [comment]
@Liwink thanks a lot for the PR!

It seems that `py.test-2.7` is still using Python 3.5.1 though. Is that the case?

**Comment by Liwink** [comment]
@sylvinus thanks!

Now it is using Python 2.7.10 ;)

```
$ py.test tests
============================================================ test session starts ============================================================
platform darwin -- Python 2.7.10 -- py-1.4.31 -- pytest-2.7.0
CircleCI total nodes: 1, this node index: 0
rootdir: /Users/Liwink/projects/mongokat, inifile:
plugins: circleci, cov, html
collected 58 items

tests/test_api.py ......
tests/test_collection.py .
tests/test_ensure_fields.py .
tests/test_hooks.py ..
tests/test_mongokit_api.py .s.............s..s.ss..s..sss.ssss...sssss..s
tests/test_readme_sample.py .
tests/test_shortnames.py .

=================================================== 39 passed, 19 skipped in 0.85 seconds ===================================================
```

File: mongokat/_bson/__init__.py [hunk 1]
```diff
@@ -641,7 +641,7 @@ def _encode_maxkey(name, dummy0, dummy1, dummy2):
 }
 
 if not PY3:
-    _ENCODERS[long] = _encode_long
+    _ENCODERS[int] = _encode_long
```
Comment (sylvinus) [inline mongokat/_bson/__init__.py:644]: this change was probably not needed?

**Comment by sylvinus** [comment]
Apart from my comment, everything looks good!

**Comment by sylvinus** [comment]
I'm merging this and adding a couple fixes afterwards. Thanks a lot for this!

**Pull request merged**